src/hooks/lib/tvl/compounder/useVaultTvlFallback.ts

```diff
@@ -19,8 +19,12 @@ export default function useVaultTvlFallback({
     useVaultTotalAssets({ vaultAddress, enabled })
   return {
     isLoading: isLoadingPricer || isLoadingTotalAssets,
-    data:
-      Number(primaryAssetPriceUsd ?? 0) *
-      (Number(totalAssets === undefined ? "0" : totalAssets.toString()) / 1e18),
+    data: {
+      usdTvl:
+        Number(primaryAssetPriceUsd ?? 0) *
+        (Number(totalAssets === undefined ? "0" : totalAssets.toString()) /
```

Comment:
`totalAssets?.toString() ?? "0"`

Reply:
I forget all the time to write it like this. Sorry about that sir.